templates/stamp/mongodb-ubuntu-install.sh

```diff
@@ -346,4 +346,4 @@ configure_db_users
 configure_replicaset
```

Comment:
i spoke offline w/ christy. i'll be rolling her changes into my change w/ my next commit